### ISSUE TITLE
File FormData parameter is not correctly converted into OAS v3

### ISSUE THREAD

**Issue by jmpascal** [Mermade/swagger2openapi#35]
Aloah,

I currently try to convert my Swagger v2 Model definition into OpenApi Specification v3.
I used https://openapi-converter.herokuapp.com/ to transform my file into v3.
After checking some values it appears parameters in formData with type file are not converted into OAS format. (see update 1)

### Checklist

- [X] Conversion: I have checked my source definition is valid OpenAPI 2.0

### Detailed Description
In my specification file I have the following parameter definition
```yaml
 parameters:
      - name: "file"
        in: "formData"
        description: "Attachment file"
        required: false
        type: "file"
```

After conversion this parameter has disappeared.
I would have expect something like

```yaml
 requestBody:
        content:
          multipart/form-data:
            schema:
              type: object
              properties:
                file:
                  description: 'Attachment File'
                  type: string
                  format: binary
```

Is it my interpretation of OAS v3 is wrong or a missing translation ?

Thanks in advance.

NB: For testing purpose:
[swagger-content.txt](https://github.com/Mermade/swagger2openapi/files/1445502/swagger-content.txt)
[oas-content.txt](https://github.com/Mermade/swagger2openapi/files/1445509/oas-content.txt)

**Update 1:**
After some investigation it appears the work is good if there's no other multipart than the file.
But
The translation is wrong if the multipart request contains a file attachment + others simple parameters (which is the case in my example)

#### Other stuff

Thanks for the excellent work !

**Comment by MikeRalphson** [comment]
Thanks for reporting this! I've made and tested a fix, which is available in swagger2openapi v2.10.6 - I'm travelling at the moment without access to my normal machine, so it may take a couple of days until the web-converter is updated.

**Comment by MikeRalphson** [comment]
web-converter updated. Please chase if this doesn't fix the issue for you.
